server/initializers/migrations/0710-video-source.ts

```diff
@@ -0,0 +1,50 @@
+import * as Sequelize from 'sequelize'
+
+async function up (utils: {
+  transaction: Sequelize.Transaction
+  queryInterface: Sequelize.QueryInterface
+  sequelize: Sequelize.Sequelize
+  db: any
+}): Promise<void> {
+  {
+    await utils.queryInterface.createTable('videoSource', {
```

Comment:
Prefer using sequelize raw query (available in logs at startup)

Reply:
May I ask why?

Reply:
To ensure we have the same table schema as a new peertube installation